Changelog.md

```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## Version 1.2.0
+* Revise fsdev initialization
+  * Removed fsdevInit/Exit
+  * Now automatically initializing fsdev state on first mount
+  * Added fsdevMountSdmc (replaces fsdevInit)
+  * Added fsdevUnmountAll (replaces fsdevExit)
+* improve usbComms support
+  * Allow using multiple interfaces.
+  * Added usbComms Ex funcs.
+  * Use RwLock.
+* hid updates
+  * Change the HidControllerLayoutType param to HidControllerType.
+  * Copy HidControllerHeader for each controller into hid state.
+  * Added hidGetHandheldMode().
+  * Added disabled hidGetControllerType().
+  * Added hidSetNpadJoyAssignmentModeSingleByDefault()
+  * Added hidSetNpadJoyAssignmentModeDual().
+* Added support for Services: fsp-ldr, fsp-pr, CSRNG, SPL, all Loader services. 
+* hid vibration support
```

Comment:
Replace this block with:
Full hid vibration support for pre-4.0.0 hid commands.